src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,129 @@
+
+* Infrastructure
+* Containers
+* Applications
+* Kubernetes clusters
+
+The Kubernetes integration of New Relic gives you full observability into the health and performance of your environment, no matter whether you run Kubernetes on-premises or in the cloud. It gives you visibility about Kubernetes namespaces, deployments, replica sets, nodes, pods, and containers. Metrics are collected from different sources:
+
+* `kube-state-metrics service` provides information about state of Kubernetes objects like namespace, replicaset, deployments, and pods (when they are not in running state).
+* `/stats/summary` kubelet endpoint gives information about network, errors, memory and CPU usage.
+* `/pods` kubelet endpoint provides information about state of running pods and containers.
+* `/metrics/cadvisor` cAdvisor endpoint provides missing data that is not included in the previous sources.
+* `/metrics` from control plane components: ETCD, controllerManager, apiServer, and scheduler.
+
+The main component of the integration is the `newrelic-infrastructure` DaemonSet. This component is divided in these other components:
+* `nrk8s-ksm`
+* `nrk8s-kubelet`
+* `nrk8s-controlplane`
+
+The first one is a deployment and the others two are DaemonSets.Each of the components has these containers:
+
+1. A container for the integration, responsible for collecting metrics.
+2. A container with the New Relic infrastructure agent, which is used to send the metrics to New Relic.
+
+<img
+  title="Diagram showing the integration installed in a 3 nodes cluster"
+  alt="Diagram showing the integration installed in a 3 nodes cluster"
+  src={kubernetesv3Integration}
+/>
+
+## Kube-state-metrics component [#nrk8s-ksm]
+
+The cluster state metrics is built on top of the OSS project [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics), which is housed under the Kubernetes organization itself. The pod in charge is the one that shares a node with the KSM deployment.
+
+This is how the KSM deployment is scraped:
+
+1. Scraping KSM to a single instance Deployment.
+2. Refactor the code and make it long-running.
+
+A specific `nrk8s-ksm` deployment uses an endpoint informer to locate the IP of the KSM pod and scrape it. The informer caches the list of informers in the cluster locally and watch for new ones.
+
+See [Bring your own KSM](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/README.md#bring-your-own-ksm) for more information about KSM supported versions.
+
+<Callout variant="important">
+  Cloud provider host tags, like `aws.ec2.*`, are no longer included in samples related to entities that are not strictly tied to a particular node: `K8sNamespaceSample`, `K8sDeploymentSample`, `K8sReplicasetSample`, `K8sDaemonsetSample`, `K8sStatefulsetSample`, `K8sServiceSample`, and `K8sHpaSample`.
+  If you need to add an attribute to these samples, use `customAttributes`.
+</Callout>
+
+## Kubelet component [#nrk8s-kubelet]
```

Comment:
Same as above: Oneliner over long internal explanation.

Reply:
I've removed it because @csongnr agrees with your comment

Reply:
once again, don't think removing it was the solution.